src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -85,8 +91,54 @@ export const survivalClinicalDataNullValueSet = new Set([
     'na',
 ]);
 
+export function sortPatientSurvivals(patientSurvivals: PatientSurvival[]) {
+    // First sort by month in asc order (smaller number to the front)
+    // Then sort by status in desc order (which means if status are '1's, then go to the front, '0's are right after '1's in the same time stamp)
+    return _.orderBy(patientSurvivals, [s => s.months, s => !s.status]);
```

Comment:
@alisman I used the way you suggested before and changed it to this because I think it's neat and still easy to understand. I can change it back to `_.orderBy(patientSurvivals, [s => s.months, s => s.status],['asc','desc']);` if you think it can lead to better understanding.